src/main/java/com/fabriccommunity/spookytime/block/TinyPumpkinBlock.java

```diff
@@ -21,18 +26,34 @@ public TinyPumpkinBlock(Settings blockSettings) {
 	
 	@Override
 	public BlockState getPlacementState(ItemPlacementContext placementContext) {
-		return this.getDefaultState().with(FACING, placementContext.getPlayerFacing().getOpposite());
+		final BlockState blockState = this.getDefaultState().with(FACING, placementContext.getPlayerFacing().getOpposite());
+		if (blockState.contains(Properties.WATERLOGGED))
```

Comment:
This is not needed as the property is added in `appendProperties`.

Reply:
There is no reason to check if this blockState contains Properties.WATERLOGGED. It always will.

Reply:
Waterlogging's now configurable, so checking is important.